appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java

```diff
@@ -192,7 +206,7 @@ public void resetMetricsSecureProperty() {
         MetricRegistry registry = getRegistry(registryName);
         Map<String, Metric> metricMap = registry.getMetrics();
         if (metricMap.containsKey(metricName)) {
-            return singletonMap(metricName, metricMap.get(metricName));
+            return Collections.singletonMap(metricName, metricMap.get(metricName));
```

Comment:
Best not to change this. "Collections." doesn't help the readability. If you want to know where something comes from you can look at the imports.